addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/ExtensionCommonlib.java

```diff
@@ -65,6 +65,7 @@ public void sessionChanged(Session session) {
         @Override
         public void sessionAboutToChange(Session session) {
             getProgressPanel().clearAndDispose();
```

Comment:
The listener needs to be added always now and the view check added to this statement.

Reply:
Okay